### DISCUSSION TITLE
Let me know your favorite restaurants!

### DISCUSSION THREAD

**Discussion by Tz-violet** [NAIST-SE/AttractingContributors#1138] [2024 Attraction Contributions]
Hi, everyone.
i want you to suggest good restaurant in my repo.
https://github.com/Tz-violet/AttractingContributors.git

after your suggestion, i will go your repo to join your project.
thank you.

**Comment by Drobotsan** [comment]
Hi, I contribute to your project.
Please accept my pull request.

Would you be kind to contribute with my project :D
https://github.com/Drobotsan/Places-in-the-world

Discussion link: https://github.com/NAIST-SE/AttractingContributors/discussions/1140

**Reply by Tz-violet** [reply]
thank you for your suggestion.
i just accepted your pull request.
and I am suggesting good place for you in your project!

**Comment by ShunHattori** [comment]
I just made a PR on your repo.
Could you contribute to my repo? Thanks in advance.
[https://github.com/ShunHattori/AttractingContributors_EssentialGadget.git](https://github.com/ShunHattori/AttractingContributors_EssentialGadget.git)
Discussion link: [#1148](https://github.com/NAIST-SE/AttractingContributors/discussions/1148)

**Comment by Marsall-k** [comment]
I have just made a PR on your Repo.
Could you contribute to mine?
https://github.com/Marsall-k/AttractingContributors-Pokemon
